src/rpc/index.ts

```diff
@@ -6,3 +6,8 @@ export async function getRpc(endpoint: string) {
   const queryClient = new QueryClient(tmClient);
   return createProtobufRpcClient(queryClient);
 }
+
+function foo() [
```

Comment:
Change to curly-brackets `{ }`
<img width="466" alt="Screen Shot 2022-07-01 at 2 54 14 PM" src="https://user-images.githubusercontent.com/2214913/176966903-a7a9802a-9ff7-497f-9475-b15a67af0f2c.png">